corehq/apps/domain/templates/domain/admin/recovery_measures_history.html

```diff
@@ -0,0 +1,61 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% block page_content %}
+    <div class="row">
```

Comment:
I think if you change this from `row` to `container` it won't get crammed up against the left-hand menu. `.row` is only for elements that you're going to fill with `.col-X-Y` divs.